client/state/selectors/get-store-referrers-by-date.js

```diff
@@ -6,10 +6,11 @@ import { find } from 'lodash';
 /**
```

Comment:
Note: I didn't move this file under `state/stats` since it makes use of woocommerce state, and is only used in woocommerce code.